modules/local/align/bwamem2_pysam.nf

```diff
@@ -16,6 +17,11 @@ process ALIGN_BWAMEM2 {
 
     script:
     def prefix = task.ext.prefix ?: "${meta.id}"
+    def cov = '0.1'
+    if (sp_richness > 150) {
+        cov = '0.01'
```

Comment:
Should we have pipeline wide parameter for this?

Reply:
That is hardcoded at the moment based on our observations in the synthetic microbial communities, but the idea is to improve this part and the pan/core modes modelling based on database features. Maybe in following releases :P

Reply:
Fair enough.
```suggestion
    # These values are hardcoded at the moment based on our observations in the synthetic microbial communities, 
    # but the idea is to improve this part and the pan/core modes modelling based on database features. 
    def cov = '0.1'
    if (sp_richness > 150) {
        cov = '0.01'
```